src/sagemaker/djl_inference/defaults.py

```diff
@@ -48,5 +48,7 @@
     "ml.g5",
     "ml.p3",
     "ml.p4",
+    "ml.p4d",
```

Comment:
Can we add "ml.p3dn" as suggested in https://github.com/aws/sagemaker-python-sdk/pull/3760/files